scripts/embedded-docs-cleanup.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+location=${1:-public}
+
+find "$location" -type f -name "*.xml" -exec test -e {} \; -exec rm -f {} +
+find "$location" -type f -name "swagger-ui*" -exec test -e {} \; -exec rm -f {} +
+rm -rf "$location"/blog
```

Comment:
does this mean we get 404s when a blog link is clicked?

Reply:
ah this was just an experiment, good catch thanks!